SEP-0002-revised.md

```diff
@@ -0,0 +1,244 @@
+
+Project Members are expected to act as role models for the community and custodians of the SunPy Project, on behalf of the community and all those who rely on SunPy.
+They will act as representatives of the SunPy Project, where necessary, including in online discussions or at official SunPy events.
+
+### Responsibilities and Rights
+
+Project Members may and should participate in formal votes on:
+
+* Adding new Project Members
+* Electing Steering Committee members
+* Appointment of the Ombudsperson
+* Removing Project Members
+* Removing Steering Committee members
+* Removing the Ombudsperson
+* Changes to this SEP
+* Other matters for which the Steering Committee believes a vote is appropriate.
+
+### Membership
+
+Project Members of the SunPy Project must demonstrate all of the following:
+
+* A good grasp of the philosophy of the SunPy Project
+* A solid track record of being constructive and helpful
+* Significant contributions to the Project's goals, in any form
+* A willingness to dedicate time to improving the Project
+* A willingness to recruit, train, and nominate new team members
+
+Project Membership acknowledges sustained and valuable efforts that align well with the philosophy and the goals of the SunPy Project.
+
+### Initial Membership
+
+To be able to elect new Project Members an initial set of Project Members are needed.
+People meeting the following criteria will be invited to form the initial group of Project Members.
+
+* Current commit access to the core repo, or any accepted affiliated package maintained by the project, and with a commit to that repository within the last 12 months.
+* Currently holds a role listed under the community roles section of the website: https://sunpy.org/project/#community-roles.
+* Is a currently serving board member.
+
+### Adding New Project Members
+
+Anyone is eligible to be nominated to be a Project member, as long as they meet the criteria laid out in the Membership section above.
+Self-nominations are encouraged.
+
+The process for adding a new Project member is as follows:
+
+1. A nomination is received by the steering committee.
+The steering committee, and the Ombudsperson check that the nomination is factually accurate and their record of contribution to the community adheres to the Code of Conduct.
+1. The candidate is then put to the Project Members.
+The name and nomination statement of the candidate will be shared with the Project Members.
+The candidate can become a Project member if five existing Project members approve the candidate and no Project Members call for an election within two weeks of the candidate being put to the Project Members.
+1. Project Members can call for an election either by posting in the discussion of the candidate or reaching out privately to the Steering Committee or the Ombudsperson.
```

Comment:
"discussion of the candidate" is really weird - I certainly wouldn't put myself forward if I was going to be "discussed"!!! If anyone wants to object I think they should do it by getting directly in touch with the steering comittee.

Reply:
Point taken, I wouldn't mind making this "inform the steering committee".

The thinking behind putting this here is because of how I expect this process to practically work (again largely stolen from astropy).

1. Steering committe gets nomination and verifies it
2. Steering committee posts to a private section of the Discourse for the project members, saying "hey this person has been nominated, do you approve or desire a vote"
3. Someone decides they want a vote and just says "vote on this one please".